README.md

```diff
@@ -1,2 +1,41 @@
-# Hello-Microverse-Review
-Review for TSE
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse project
+
+> set up of a "Hello world" repository. No complex coding
+
+![screenshot]()
```

Comment:
please remember to delete this line if you don't have a screenshot to make your readme more professional, avoid empty lines and qualifiers you don't use.

Reply:
Hi @vic778,
Thank you so much for your kind review. I will make the changes as soon as possible